release.sh

```diff
@@ -62,3 +66,16 @@ gcutil push fe tmp/$CHANNEL /var/www/install.sandstorm.io
 
 gcutil ssh smalldemo sudo service sandstorm update
 gcutil ssh alpha sudo service sandstorm update dev
+
+echo "**** Tagging this commit ****"
+
+# The git tag stores the version number as a normal-looking version
+# number, like 0.75 for build 75 within branch 0, or 2.121 for build
+# 121 within branch 2.
+
+BUILD_MINOR="$(( $BUILD % 1000 ))"
+DISPLAY_VERSION="${BRANCH_NUMBER}.${BUILD_MINOR}"
+TAG_NAME="sandstorm-${DISPLAY_VERSION}"
```

Comment:
"sandstorm" seems redundant since the tag is already in the Sandstorm repository. Maybe just "v$DISPLAY_VERSION"?